_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the letter spacing can be adjusted by the user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
```

Comment:
I'm not a fan of the way this is formulated. The first part is too broad a description of what this rule actually does. I think 
```suggestion
  This rule checks that the style attribute is not used to prevent increasing letter spacing by using `!important`, except if the 0.12 time the font size.
```

Reply:
I prefer keeping the description generic rather than putting out actual metric like 0.12 in the description. If others feel the same about this. I am willing to change.

Am I right in thinking that based on this [comment](https://github.com/act-rules/act-rules.github.io/pull/1412#discussion_r484744367), you are not looking to approve this rule?

Reply:
The reason I think this should change is because the first part of it is so generic, it might as well be used on other rules. This doesn't set the rule apart from other rules that could be written for this SC.

Reply:
This description seems a little "chatty" to me. What is the problem with a more succinct rewording along the lines of what @WilcoFiers is proposing?